certbot/tests/ocsp_test.py

```diff
@@ -71,21 +82,18 @@ def test_ocsp_revoked(self, mock_run, mock_determine):
         self.assertEqual(self.checker.ocsp_revoked("x", "y"), False)
         self.assertEqual(mock_run.call_count, 2)
 
+    def test_determine_ocsp_server(self):
+        tmpdir = tempfile.mkdtemp()
+        try:
+            cert_path = os.path.join(tmpdir, 'cert.pem')
+            with open(cert_path, 'w') as file_handler:
+                file_handler.write(google_certificate)
```

Comment:
Rather than writing out the certificates here and in `OSCPTestCryptography.setUp` below, we usually put stuff like this in the [testdata](https://github.com/certbot/certbot/tree/master/certbot/tests/testdata) directory. Once it's there, you can use `certbot.tests.util.load_vector` with the filename to get the path to the certificate on disk.

Reply:
Indeed. Way better to store and manipulate test assets. It is `vector_path` by the way to get the path, `load_vector` returns its content.